src/infrastructure/migrations/1729516821137-AddChainIdToVotingRound.ts

```diff
@@ -0,0 +1,33 @@
+import type { MigrationInterface, QueryRunner } from 'typeorm';
+import appSettings from '../../appSettings';
+
+export class AddChainIdToVotingRound1729516821137
+  implements MigrationInterface
+{
+  name = 'AddChainIdToVotingRound1729516821137';
+  schema = appSettings.network.name;
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Add the column 'chainId' without the NOT NULL constraint.
+    await queryRunner.query(`
+        ALTER TABLE "${this.schema}"."VotingRounds" ADD COLUMN "chainId" integer;
+      `);
+
+    // Update existing rows to set the chain ID to 1. At the point of migration we only have data for the Ethereum mainnet.
+    await queryRunner.query(`
+        UPDATE "${this.schema}"."VotingRounds" SET "chainId" = ${appSettings.chainId};
```

Comment:
shouldn't chainId actually be hardcoded to 1 here?

Reply:
Within the context of this PR we're still using multiple schemas so it's better to set this new column to the real chain ID. When we move to one schema with the [chain agnostic changes](https://github.com/drips-network/multiplayer/pull/20), there, it will have the chain ID set to 1, by design.